runtime/interpreter/value_string.go

```diff
@@ -108,6 +110,59 @@ func stringFunctionFromCharacters(invocation Invocation) Value {
 	return NewUnmeteredStringValue(builder.String())
 }
 
+func stringFunctionSplit(invocation Invocation) Value {
+	s, ok := invocation.Arguments[0].(*StringValue)
+	if !ok {
+		panic(errors.NewUnreachableError())
+	}
+
+	inter := invocation.Interpreter
+
+	separator, ok := invocation.Arguments[1].(*StringValue)
+	if !ok {
+		panic(errors.NewUnreachableError())
+	}
+
+	split := strings.Split(s.Str, separator.Str)
+
+	var index int
+	count := len(split)
+
+	return NewArrayValueWithIterator(
+		inter,
+		NewVariableSizedStaticType(inter, PrimitiveStaticTypeString),
```

Comment:
Refactor out of the function, it's "constant"

Reply:
Extracted into a global variable but had to pass `nil` in the memory guage.